Dockerfile.template

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:experimental
+
+ARG PYTHON_VERSION
+
+FROM ${PYTHON_VERSION}
+
+ENV TOX_WORK_DIR=/tmp/.tox
```

Comment:
Need this for docker on mac; docker on mac's (and on windows to be honest) volume-mapped-IO performance is awful, to make it a little more tolerable we move the tox stuff to the docker machine's "disk" increasing performance 10-fold!

Reply:
Not relevant anymore